src/Apps/Order/Routes/Review/index.tsx

```diff
@@ -80,12 +80,13 @@ export class ReviewRoute extends Component<ReviewProps, ReviewState> {
       },
     ],
   }))
-  async onSubmit() {
+  async onSubmit(setupIntentId?: null) {
```

Comment:
is the `?` new Typescript thing? never seen it before!

Reply:
no it's been around forever. Just means the argument is optional.